components/button.json

```diff
@@ -3,19 +3,23 @@
     "primary":{
       "#normal":{
         "background":"@theme-button-primary-normal",
-        "text": "@theme-text-primary-inverted"
+        "text": "@theme-text-primary-inverted",
+        "border": "@theme-button-primary-normal"
```

Comment:
Primary buttons don't ever show a border in any state - why have you added these?

Reply:
Ah, you're having a border the same colour as the background. This probably needs to be discussed with the other teams implementing it to check that it won't cause issues. @InteractiveTimmy thoughts from the web POV?